File: Web/EvaMittal_EvaMittal_2125cs1187_2/GettingStarted/IsANumberPrime.java
import java.util.*;
public class IsANumberPrime {
    public static void main(String[] args) {
        Scanner scn = new Scanner(System.in);
    
         // write ur code here
         int t = scn.nextInt();
         int i,j,f;
         for(i=1;i<=t;i++)
         {
             f=0;
         int n= scn.nextInt();
         for(j=2;j<=n/2;j++)
         {
             if(n%j==0)
             {
                 f=1;
                 break;
             }
         }
         if(f==0)
         System.out.println("prime");
         else
         System.out.println("not prime");
         }
    
     }
}

File: Web/Divyanshu-Vareshney_divyanshuvarshney_2125it1075_2/getting started/Is A Number Prime.cpp
#include <iostream>
using namespace std;
int main(int argc, char **argv){
    int t;
    cin >> t;

    //write your code here
    for(int i=1;i<=t;i++){
        int n;
        int c=0;
        cin>>n;
        for(int j=1;j<=n;j++){
            if(n%j==0){
                c++;
            }
        }
        if(c==2){
            cout<<"prime"<<"\n";
        }
        else{
            cout<<"not prime"<<"\n";
        }
    }
    
}